recipe/patches/0001_remove_protobuf_requirement.patch

```diff
@@ -0,0 +1,80 @@
+diff --git a/.azure-pipelines/linux-conda-CI.yml b/.azure-pipelines/linux-conda-CI.yml
+index 6675d37..590b406 100644
+--- a/.azure-pipelines/linux-conda-CI.yml
++++ b/.azure-pipelines/linux-conda-CI.yml
```

Comment:
I think only `pyproject.toml` has to be modified. You don't need to modify `requirements.txt` and files in `.azure-pipelines`.